braindecode/preprocessing/preprocess.py

```diff
@@ -130,34 +144,73 @@ def preprocess(concat_ds, preprocessors):
         assert hasattr(elem, 'apply'), (
             'Preprocessor object needs an `apply` method.')
 
```

Comment:
Please add a check at the beginning of `preprocess` that crashes when `overwrite=False` and directory contains files. We don`t want to wait for a long time for the preprocessing to finish until it crashes when attempting to save.

Reply:
Great idea!